generic/default/go-client/cmd/client.go

```diff
@@ -57,7 +56,7 @@ func main() {
 	//callQueryAll(dubboRefConf)
 
 	// generic invocation samples using hessian serialization on Triple protocol
-	tripleRefConf := newRefConf("org.apache.dubbo.samples.UserProviderTriple", tpconst.TRIPLE)
+	tripleRefConf := newRefConf("org.apache.dubbo.UserProviderTriple", tpconst.TRIPLE)
```

Comment:
Why remove `.sample` from the code? It may cause something conflicts with dubbo-java in some cases, cuz they are using `org.apache.dubbo.xxx` as package name. cc @LaurenceLiZhixin

Reply:
I forgot where conflict occur. It, however, has happened.

Reply:
the class's path in go and java is different, I will add sample word.